JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java

```diff
@@ -1000,6 +1013,26 @@ public void setElevation(List<StaticPointSource> pointSources, List<Building> bu
 
     }
 
+    List<byte[]> getScopeElevation(Polygon scope, int srid) {
+
+        String sql = String.format("SELECT ST_AsGDALRaster(rast, 'GTiff') AS rData FROM elevation " +
+                "WHERE ST_Intersects(rast, ST_Transform(ST_GeomFromText('%s',4326),%d));", scope.toText(), srid);
+        sql = sql.replace("wkt", scope.toText());
```

Comment:
where does wkt come from?

Reply:
I will delete that line. It is not required.

Reply:
It has been removed.